code/modules/clothing/under/miscellaneous.dm

```diff
@@ -243,6 +243,15 @@
 	icon_state = "white_suit"
 	item_state = "white_suit"
 	item_color = "white_suit"
+	whiteness = 0.5
+
+/obj/item/clothing/under/burial
+	name = "burial garments"
+	desc = "Traditional burial garments from the early 22nd century."
+	icon_state = "burial"
+	item_state = "burial"
+	item_color = "burial"
+	whiteness = 1

```

Comment:
I don't like that you are adding special vars just for devil side antag. Make it a list path = value somewhere in devil's code.